Airports/EGTU/Regions

```diff
@@ -0,0 +1,279 @@
+; Start Dunkeswell (EGTU) Regions
+; - Runway
```

Comment:
Can you follow the conventions in other files please?
So instead of ; - Terminal, just ;Terminal

See example here: https://github.com/VATSIM-UK/UK-Sector-File/blob/master/Airports/EGKK/SMR/Regions.txt